tests/secure_verify/host/evidence.cpp

```diff
@@ -168,21 +168,39 @@ bool read_binary_file(
 oe_result_t verify_oe_evidence(
     oe_enclave_t* enclave,
     const oe_uuid_t* foramt_id,
-    const char* evidence_filename)
+    const char* evidence_filename,
+    const char* endorsement_filename)
 {
     oe_result_t result = OE_UNEXPECTED;
     size_t evidence_file_size = 0;
     uint8_t* evidence_data = NULL;
 
+    size_t endorsement_file_size = 0;
+    uint8_t* endorsement_data = NULL;
+
     if (!read_binary_file(
             evidence_filename, &evidence_data, &evidence_file_size))
     {
         result = OE_INVALID_PARAMETER;
         goto done;
     }
 
+    if (endorsement_filename &&
+        !read_binary_file(
+            endorsement_filename, &endorsement_data, &endorsement_file_size))
+    {
+        result = OE_INVALID_PARAMETER;
+        goto done;
+    }
+
     verify_plugin_evidence(
-        enclave, &result, foramt_id, evidence_data, evidence_file_size);
+        enclave,
+        &result,
+        foramt_id,
+        evidence_data,
+        evidence_file_size,
+        endorsement_data,
+        endorsement_file_size);
```

Comment:
Is it possible to verify that we got SGX_RELAUNCH_ADVISED for the v5 quote?

Reply:
Checked and the quote is already v5

Reply:
I mean programmatically in the tests :)